tests/antithesis/test-template/Dockerfile.client

```diff
@@ -13,7 +13,7 @@ RUN apt-get install -y python3-etcd3 python3-numpy python3-protobuf python3-file
 RUN pip install antithesis cffi --break-system-packages
 
 # Copy your entrypoint script
-COPY ./entrypoint/entrypoint.py /entrypoint.py
+COPY ./entrypoint/entrypoint.go /entrypoint.go
```

Comment:
You need to compile the  go file the same way you compile `serial_driver_delete_keys` below